Essentials/build.gradle

```
@@ -8,6 +8,10 @@ dependencies {
 
     api 'org.bstats:bstats-bukkit:1.8'
 
+    api 'org.spongepowered:configurate-yaml:4.0.0'
```

```suggestion
    api 'org.spongepowered:configurate-yaml:4.1.1'
```

4.1.0 and 4.1.1 have a few fixes - might be worth updating?